tests/dtm/test_single_process_restart.py

```diff
@@ -108,6 +106,19 @@ def setup_method(self):
         self.object_name = f"dps-obj-{self.random_time}"
         self.deploy = False
         self.iam_user = dict()
+        self.log.info("Create IAM user with name %s", self.s3acc_name)
+        resp = self.rest_obj.create_s3_account(acc_name=self.s3acc_name,
+                                               email_id=self.s3acc_email,
+                                               passwd=S3_CFG["CliConfig"]["s3_account"]["password"])
+        assert_utils.assert_true(resp[0], resp[1])
+        self.access_key = resp[1]["access_key"]
+        self.secret_key = resp[1]["secret_key"]
+        self.iam_user = {'s3_acc': {'accesskey': self.access_key, 'secretkey': self.secret_key,
+                                    'user_name': self.s3acc_name}}
+        self.s3_test_obj = S3TestLib(access_key=self.access_key, secret_key=self.secret_key,
+                                     endpoint_url=S3_CFG["s3_url"])
+        self.dtm_obj = DTMRecoveryTestLib(access_key=self.access_key, secret_key=self.secret_key)
```

Comment:
won't this need retry count set?

Reply:
For m0d restart, no retries from client side are expected, so keeping it default None.